src/tests/functional/shared_test_classes/include/shared_test_classes/single_op/fake_quantize.hpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <tuple>
+#include <vector>
+#include <string>
+#include <memory>
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+
+// seed selected using current cloc time
+#define USE_CLOCK_TIME 1
+// seed started from default value, and incremented every time using big number like 9999
+#define USE_INCREMENTAL_SEED 2
+
+/**
+ * redefine this seed to reproduce issue with given seed that can be read from gtest logs
+ */
+#define BASE_SEED   123
+#define OV_SEED 123
```

Comment:
Hm. Looks like hardcode. Let's ask fake quantize @eshoguli . @eshoguli What do you think about this tests? It it correct or should be removed?

Reply:
I removed all hardcode because it doesn't work indeed